source/mas_piloot_steekproefdesign/zzz_references_and_appendix.Rmd

```diff
@@ -4,6 +4,89 @@
 INBOmd::references(appendix = TRUE)
 ```
 
-# Eerste hoofdstuk van de bijlage
+# Bijlagen {#bijlagen}
 
-Inhoud van de eerste bijlage
+## Traceerbaarheid van de steekproefkaders en steekproeven besproken in dit rapport {#hashes-steekproef}
+
+De steekproefkaders en steekproeven staan als *TSV*-bestanden onder versiecontrole in de GitHub repository.
+De combinatie van data en metadata hashes kunnen de datasets op unieke manier identificeren (en dus bijvoorbeeld onderscheiden van oudere versies).
+De onderstaande tabel geeft de hashes van de finale datasets.
```

Comment:
Kan je hier nog een zinnetje bijzetten dat deze hashes berekend worden wanneer je de bestanden opslaat en wegschrijft met git2rdata::write_vc package? De hashes van data (data_hash) en metadata (hash) staan in het bijhorende .yml bestand.

Reply:
Ja goed idee. Het is aangepast.